api/deps.edn

```diff
@@ -71,7 +73,7 @@
                                                 :exclusions [ch.qos.logback/logback-classic]}
                  org.liquibase/liquibase-core  {:mvn/version "4.0.0"
                                                 :exclusions [ch.qos.logback/logback-classic]}}
-   :extra-paths ["resources" "test"]
+   :extra-paths ["../derived/api/resources" "resources" "test"]
```

Comment:
I suspect we don't need to include the resources folders here because they're already included in the main paths section

Reply:
You're right.